### ISSUE TITLE
Timeline date handling chokes on early Julian dates

### ISSUE THREAD

**Issue by erikmaarten** [AnalyticalGraphicsInc/cesium#4544]
When zooming out on the timeline so the start date is changed to an negative/invalid ISO8601 date, the `JulianDate.fromIso8601` function throws. 

The following code in `Timeline.js` is where the error occurs:

``` js
if (duration > 315360000) { // 3650+ days visible, epoch is start of the first visible century.
    epochJulian = JulianDate.fromIso8601(JulianDate.toDate(startJulian).toISOString().substring(0, 2) + '00-01-01T00:00:00Z');
}
```

Perhaps when zooming, we could check whether the from date is smaller than some minimum date and in that case simply set the timeline from date to the minimum value?

Which year range could be considered a reasonable default?

/cc @hpinkos @mramato

**Comment by mramato** [comment]
`ISO 8601` min/max values are defined in constants at https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/Iso8601.js#L28, so you could probably use those to check.

**Comment by erikmaarten** [comment]
Might also fix for #4425.
